package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "crds-styles",
-  "version": "3.2.4",
+  "version": "3.2.5",
```

Comment:
@lukemoderwell We're going to be releasing either 3.3.0 or 4.0.0 at the end of this sprint, as we'll be releasing new styles for shared header, footer, and snail trail skeleton blocks, and we'll remove some of the old styles.

You don't need to change anything here, but just to let you know.